platform-hub-api/app/models/ability.rb

```diff
@@ -53,6 +53,16 @@ def initialize(user)
     end
 
 
+    # Docker repos in projects
+
+    can :read_docker_repos_in_project, Project do |project|
+      can_participate_in_project project, user
+    end
+    can :administer_docker_repos_in_project, Project do |project|
+      can_participate_in_project project, user
```

Comment:
This should be `can_administer_project` ?

Reply:
I think the [consensus in #hub_ecr](https://hod-dsp.slack.com/archives/CDMFKN5TJ/p1541601224025900) was that any one in a project can create or delete repos.